cmd/derper/bootstrap_dns.go

```diff
@@ -35,6 +37,12 @@ var (
 	unpublishedDNSMisses = expvar.NewInt("counter_bootstrap_dns_unpublished_misses")
 )
 
+func init() {
+	expvar.Publish("counter_bootstrap_dns_queried_domains", expvar.Func(func() any {
+		return fmt.Sprintf("%d", bootstrapLookupMap.Len())
```

Comment:
Optional, but... `expvar.Func` works by formatting the returned value using JSON, so you should be able to return `bootstrapLookupMap.Len()` directly here, without the `fmt.Sprintf`.

Reply:
Done.